presto-mongodb/src/test/java/com/facebook/presto/mongodb/TestMongoIntegrationSmokeTest.java

```diff
@@ -132,6 +135,7 @@ public void testInsertWithEveryType()
         assertEquals(row.getField(4), true);
         assertEquals(row.getField(5), LocalDate.of(1980, 5, 7));
         assertEquals(row.getField(6), LocalDateTime.of(1980, 5, 7, 11, 22, 33, 456_000_000));
+        assertEquals(row.getField(7), LocalTime.of(11, 22, 33, 456_000_000));
```

Comment:
can you test the same with the old code to ensure that it is the same, since this path was not getting tested previously.

Reply:
I validated this test before and after removing the Joda-time library. You think I should split this into two commits/PRs for clarity?

Reply:
I think that is fine since this test is related to this refactoring.